packages/driver-interface/src/services/messaging.ts

```diff
@@ -70,16 +71,23 @@ export const sendPickupInstruction = async (
       : await getAddressFromCoordinate({ ...firstBooking.delivery })
 
   const message = (instructionGroup.length === 1
-    ? `🎁 Hämta paket "${helpers
-        .getLastFourChars(instructionGroup[0].id)
-        .toUpperCase()}" vid [${pickup}](${getDirectionsUrl(
+    ? `🎁 Din nästa destination är [${pickup}](${getDirectionsUrl(
```

Comment:
```suggestion
    ? `🎁 Ditt nästa stopp är [${pickup}](${getDirectionsUrl(
```
Stopp känns tydligare än destination.